src/components/fields/input-field.vue

```diff
@@ -13,20 +13,30 @@
         </template>
       </template>
 
-      <input
-        v-bind="computedAttrs"
-        :type="type"
-        :value="modelValue"
-        :style="inputFieldStyles"
-        :disabled="disabled"
-        :tabindex="tabindex"
-        :min="minValue"
-        class="input-field__input"
-        autocomplete="off"
-        autocorrect="off"
-        autocapitalize="off"
-        spellcheck="false"
-      >
+      <div class="input-field__wrapper">
+        <div class="leading-icon">
+          <span class="iconLeading"></span>
+        </div>
```

Comment:
This functionality should be implemented with slots. Right now it does literally nothing